runtime/sam/expr/function/types.go

```diff
@@ -87,11 +86,10 @@ func (i *Is) Call(_ super.Allocator, args []super.Value) super.Value {
 	}
 	var typ super.Type
 	var err error
-	if zvTypeVal.IsString() {
-		typ, err = sup.ParseType(i.sctx, string(zvTypeVal.Bytes()))
-	} else {
-		typ, err = i.sctx.LookupByValue(zvTypeVal.Bytes())
+	if zvTypeVal.Type().ID() != super.IDType {
+		return i.sctx.WrapError("is(): type value argument expected", zvTypeVal)
```

Comment:
We don't usually include parentheses in these error strings.
```suggestion
		return i.sctx.WrapError("is: type value argument expected", zvTypeVal)
```